package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@reduxjs/toolkit": "^1.9.3",
+    "@types/react-icons": "^3.0.0",
```

Comment:
Please also remove this package. Icons are already added in a previous P.R, and they come with their own type definitions. 

P.S. As a word of advice, don't install type definitions as dependencies. They are always devDependencies.